pkg/migration/controllers/clusterpair.go

```diff
@@ -229,6 +231,30 @@ func (c *ClusterPairController) cleanup(clusterPair *stork_api.ClusterPair) erro
 	if !skipDelete && clusterPair.Status.RemoteStorageID != "" {
 		return c.volDriver.DeletePair(clusterPair)
 	}
+
+	// Delete the backuplocation and secret associated with clusterpair as part of the delete
+	if backuplocationName, ok := clusterPair.Spec.Options["backuplocation"]; ok {
+		bl, err := storkops.Instance().GetBackupLocation(backuplocationName, clusterPair.Namespace)
+		if err != nil && !errors.IsNotFound(err) {
+			logrus.Errorf("fetching backuplocation %s in ns %s failed: %v", backuplocationName, clusterPair.Namespace, err)
```

Comment:
nit...We can return from here and not have err==nil condition check in the next statement to make it cleaner. Something like if   err != nil {if !IsNotFound(err) -> Log error, return nil}

Reply:
Done

Reply:
Thanks for doing this change